traffic_stats/grafana/traffic_ops_server.js

```diff
@@ -13,1368 +13,1359 @@
      limitations under the License.
 */
 
-/* global _ */
-
 /*
  * Scripted dashboard for traffic ops.
  *
  * Based on the grafana scripted.js script (which is ASF 2.0 licensed).
  */
 
-
-
-// accessible variables in this scope
-var window, document, ARGS, $, jQuery, moment, kbn;
+'use strict';
 
 // Setup some variables
 var dashboard;
 
-// All url parameters are available via the ARGS object
+// All URL parameters are available via the ARGS object
 var ARGS;
 
-// Intialize a skeleton with nothing but a rows array and service object
+// Intialize a skeleton with nothing but a rows array and service object,
+// and setting default time and refresh interval.
 dashboard = {
-  rows : [],
-};
-
-
-// Set default time
-// time can be overriden in the url using from/to parameters, but this is
-// handled automatically in grafana core during dashboard initialization
-dashboard.time = {
-  from: "now-24h",
-  to: "now"
+  refresh: "30s",
+  rows: [],
+  // time can be overridden in the URL using from/to parameters, but this is
+  // handled automatically in grafana core during dashboard initialization
+  time: {
+    from: "now-24h",
+    to: "now"
+  }
 };
 
-var which = 'argName';
+let which = 'argName';
 
-if(!_.isUndefined(ARGS.which)) {
+if(ARGS.which !== undefined) {
```

Comment:
Nit: Since the whole file is reformatted now, `if(` should be `if (`.

Reply:
Oh god it is isn't it?

Reply:
It looks like the change that blew up the diff was removing the curly braces around `dashboard.rows.push()`. After adding those back and reformatting, it's way more straightforward to see what changed.

Reply:
Yes... but also idk why that nested block was there in the first place. I'll have to put it back in, I suppose.

Reply:
The brackets were added in Comcast/traffic_control#592, before which `traffic_ops_server.js` was just [the `else if ( type == "server" ) {}` clause](https://github.com/apache/trafficcontrol/blob/958e32a0f6/traffic_stats/grafana/traffic_ops_scripted.js#L376-L523) of a big if/else statement in `traffic_ops_scripted.js`, so there is no reason to keep them. But yeah, better to save removing the brackets from this and the other scripted dashboards for a separate PR.